vendor/vendor.json

```diff
@@ -73,7 +73,7 @@
 			"tree": true
 		},
 		{
-			"checksumSHA1": "g+RGhbXqsyBOtKpY08hPDQOeuqI=",
+			"checksumSHA1": "Di7Kk9jBU/wKAU60owZtUo/NqYE=",
```

Comment:
you should reset this change. checksum changes don't need to be committed and just cause merge conflicts